src/main/java/com/anonymous/privacytool/service/impl/UserServiceImpl.java

```diff
@@ -44,6 +61,44 @@ public void processForgotPassword(String email) {
         }
     }
 
+    @Override
+    public ResponseEntity<GenericResponse> signUpUser(User user) throws Exception {
+        boolean alreadyRegister=checkIfUserIsAlreadyRegistered(user);
+
+        if(alreadyRegister==true){
+            throw new Exception("User already registered");
+        }
+
+        userRepository.save(user);
```

Comment:
Validate and create fresh user object to save.
User custom request pojo instead of User DB Model.
This will save just anything.